include/mbedtls/pk.h

```diff
@@ -245,6 +247,12 @@ typedef struct mbedtls_pk_context {
 #if defined(MBEDTLS_PSA_CRYPTO_C)
     mbedtls_svc_key_id_t MBEDTLS_PRIVATE(priv_id);      /**< Key ID for opaque keys */
 #endif /* MBEDTLS_PSA_CRYPTO_C */
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    uint8_t MBEDTLS_PRIVATE(pk_raw)[MBEDTLS_PK_MAX_EC_PUBKEY_RAW_LEN];  /**< Raw public key   */
```

Comment:
As for the `pub_raw` renaming, I just found out that actually `mbedtls_x509_buf pk_raw;` is a field of the `struct mbedtls_x509_crt` so it's even more suggested to rename this field to `pub_raw` to avoid confusion!